UI/DragNDrop.cpp

```diff
@@ -32,8 +37,8 @@ void DragNDrop::update()
 	if (m_isDragging)
 	{
 		m_attachedObject->getTransform().place2D(
-			input::InputManager::getInstance()->getMouseXPos() - 5,
-			input::InputManager::getInstance()->getMouseYOpenGLPos() - getTransform().getScale2D().y + 5
+			input::InputManager::getInstance()->getMouseXPos() - 25,
+			input::InputManager::getInstance()->getMouseYOpenGLPos() - getTransform().getScale2D().y + 25
```

Comment:
I'm not sure what this change does, but if it is a visual change that is fine -- just adding comment so you are aware of it

Reply:
Visual Change